### PR TITLE
Improve Challenge Screen

### PR DESCRIPTION
**Pull request by chihsuan**

### CODE REVIEW

**Comment by TseHang** [comment]
Why it names `Challenge`.....?

File: src/screens/ChallengeScreen.tsx [hunk 1]
```diff
@@ -1,103 +1,133 @@
...
+            <View style={myStyles.descriptionWrapper}>
+              <Text style={myStyles.description}>
+                is requesting information below
+              </Text>
+            </View>
+
+            <FieldRow
+              title="Date of purchase："
+              description={getToday()}
+              style={myStyles.row}
```
Comment (TseHang) [inline src/screens/ChallengeScreen.tsx]: My ts-linter will print error like this,


Should we need to fix this ?

File: src/screens/ChallengeScreen.tsx [hunk 2]
```diff
@@ -1,103 +1,133 @@
...
+            />
 
-          <FieldRow
-            title="Push notification"
-            description="Allow"
-          />
+            <FieldRow
+              title="Name："
+              description={this.props.identity.username || ''}
+              style={myStyles.row}
```
Comment (TseHang) [inline src/screens/ChallengeScreen.tsx]: My ts-linter will print error like this,


Should we need to fix this ?

File: src/screens/ChallengeScreen.tsx (hunk 1 above)
Comment (TseHang) [reply src/screens/ChallengeScreen.tsx]: Maybe add `style` props in `FiledRow` ?

Comment (TseHang) [reply src/screens/ChallengeScreen.tsx]: Oh I see you fix it in another [pr](https://github.com/visuallylab/BiiMe/pull/9/files)

File: src/screens/ChallengeScreen.tsx (hunk 2 above)
Comment (TseHang) [reply src/screens/ChallengeScreen.tsx]: Oh I see you fix it in another [pr](https://github.com/visuallylab/BiiMe/pull/9/files) 👍

**Comment by chihsuan** [comment]
@TseHang: Good question. You can refer to the DID doc in our documents. The name is inherited from TangleID.

**Pull request merged**